src/pages/landing/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Features from './components/features'
+import Hero from './components/hero'
+
+function Landing() {
+  return (
```

Comment:
I'd remove the Landing component and simply place the Hero and Features component directly on the index page so we make things more flatten